Imagr/Utils.py

```diff
@@ -516,7 +517,32 @@ def launchApp(app_path):
 
 
 def get_hardware_info():
-    '''Uses system profiler to get hardware info for this machine'''
+    
+    '''
+    system_profiler is not included in a 10.13 NetInstall NBI, therefore a new method of getting serial numer and model identifier is required
```

Comment:
Please remove the extra line break on line 520 and switch the Doc string comment to use triple double instead of triple single quote. This matches the style inside this file.